schemas/CreateProductInput.json

```
@@ -0,0 +1,214 @@
+{
+  "type": "object",
+  "required": ["product_overview", "product_lifecycle_policies", "promotional_policies", "post_promotional_policies"],
+  "additionalProperties": false,
+  "properties": {
+    "effective_at": {
+      "type": "string",
+      "format": "date-time",
+      "description": "The `Date-Time` as of which this product is effective on the server.",
+      "example": "2016-11-27T13:19:56+00:00"
+    },
+    "external_product_id": {
+      "type": "string",
+      "description": "A unique external ID that may be used interchangeably with the Canopy-generated product ID",
+      "example": "95fea111-d121-40f8-9e8e-a6c335e48a45"
+    },
+    "product_overview": {
+      "type": "object",
+      "required": ["product_name", "product_type", "product_short_description", "product_long_description"],
+      "description": "A series of static fields about the product.",
+      "properties": {
+        "product_name": {
+          "type": "string",
+          "description": "Name of Product, i.e. BNPL 24 Month Standard.",
+          "example": "BNPL 24 Month Standard"
+        },
+        "product_type": {
+          "type": "string",
+          "description": "The Type of Product",
+          "default": "REVOLVING",
+          "enum": [
+            "REVOLVING",
+            "INSTALLMENT",
+            "MIXED_RATE_INSTALLMENT",
+            "DEFERRED_INSTALLMENT",
+            "FIXED_RATE_INSTALLMENT"
+          ],
+          "example": "REVOLVING"
+        },
+        "product_short_description": {
+          "type": "string",
+          "description": "A short description of the product ranging from 0 - 60 characters.",
+          "example": "12mm3apr."
+        },
+        "product_long_description": {
+          "type": "string",
+          "description": "Description of the Product ranging from 0 - 1000.",
```

```suggestion
          "description": "Description of the product ranging from 0 - 1000.",
```

Should we continue the capitalization of entities in Canopy or just lowercase them all?